src/matrixHelpers/FormioScript/scriptGitBridge.js

```diff
@@ -0,0 +1,44 @@
+import { createScripts } from ".";
+
+const version = window.formioQldCdnVersion || "248740";
+
+const scripts = [
+  {
+    type: "script",
+    src: `https://www.qld.gov.au/__data/assets/git_bridge/25/${version}/formio.full.js`,
```

Comment:
25 needs to be 0025 else it breaks
https://www.qld.gov.au/__data/assets/git_bridge/0025/248740/formio.full.js
vs
https://www.qld.gov.au/__data/assets/git_bridge/25/248740/formio.full.js